### ISSUE TITLE
`viz` command does not work on Windows

### ISSUE THREAD

**Issue by pkalita-lbl** [INCATools/ontology-access-kit#477]
On Windows, if I run a basic `viz` command from the help, I get the following error:

```shell
> runoak -i ubergraph: viz CL:4023094
...
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\tmp\tmpf4ykx4hm'
```
 
The proximate cause of this error is that `C:\tmp` does not exist by default on Windows. If I create that directory manually and run the same command I get a similar but more cryptic error:

```shell
> runoak -i ubergraph: viz CL:4023094
...
FileNotFoundError: [Errno 2] The system cannot find the file specified
```

I think this points to a deeper issue where the the `viz` command is managing temporary files in a way that's not totally cross-platform friendly.

**Comment by shawntanzk** [comment]
Just got my computer wiped and had to re-download oaklib so it was updated to latest version, seems the problem is back. I have checked that the changes in the PR connected to this is in my system, so not sure what is going on now.  Here is the whole terminal input/output. 

Thanks :) 

```
(venv) PS C:\Users\TZKS> runoak -i sqlite:obo:fbbt viz FBbt:00004751 -p i
node:fs:602
  handleErrorFromBinding(ctx);
  ^

Error: ENOENT: no such file or directory, open '/tmp/foo.dot'
    at Object.openSync (node:fs:602:3)
    at writeFileSync (node:fs:2334:35)
    at file:///c:/Users/TZKS/venv/Scripts/node_modules/obographviz/bin/og2dot.js:66:9
    at Array.forEach (<anonymous>)
    at file:///c:/Users/TZKS/venv/Scripts/node_modules/obographviz/bin/og2dot.js:52:14
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:228:24)
    at async loadESM (node:internal/process/esm_loader:40:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: '/tmp/foo.dot'
}

Node.js v20.5.0
The system cannot find the file C:\Users\TZKS\AppData\Local\Temp\tmp74o76uy0.png.
```

**Comment by pkalita-lbl** [comment]
Hi @shawntanzk sorry for not getting back to you sooner. It looks like you're actually seeing [this bug](https://github.com/INCATools/obographviz/issues/29) in `obographviz` which has been fixed but for some reason didn't get published. I've just released version 0.4.3 of https://www.npmjs.com/package/obographviz. Try upgrading to the latest version of that (likely `npm install -g obographviz` should be enough) and see if that fixes the issue for you.
